src/Wrappers.dfy

```diff
@@ -63,6 +63,13 @@ module Wrappers {
       Failure(this.error)
     }
 
+    function method MapFailure<NewR>(reWrap: R -> NewR): Result<T, NewR>
+    {
+      match this
+        case Success(s) => Success(s)
```

Comment:
Can you align the `case` with the `match` so that it is consistent with the rest of the file?